app/views/posts/show.html.erb

```diff
@@ -2,17 +2,22 @@
   <div class="post_desc">
     <div class="unit">
       <h2><%= @post.title %> by <%= @post.author.name%></h2>
-      <h4>Comments:<%= @post.comments_counter%>,  Likes:<%= @post.likes_counter%></h4>
+      <h4>Comments:<%= @post.comments_counter%></h4>
+      <h4>
+        <%= button_to "Like",
+        user_post_likes_path(user_id: @current_user.id, post_id: @post.id),
+        method: :post, class: "like-button" %> <%= @post.likes_counter %> </h4>
     </div>
     <p class="P_text"><%= @post.text%></p>
   </div>
 
   <div class="com_on_Post">
-  <h3>comments:</h3>
+  <h3>comments</h3>
     <ul>
       <% @post_comments.each do |comment| %>
         <li><%= comment.author.name %>: <%= comment.text %></li>
       <% end %>
     </ul>
+    <%= link_to "Add comment", new_user_post_comment_path(user_id: @current_user.id, post_id: @post.id) %>
```

Comment:
On the **Post Index** page, it should only display the recent 5 comments on the post. You shoud be using `recent_comments` method in the **Post** model to do this job. However, in your case, it is displaying all the comments of a post. Kindly fix this issue. ⬇️
![image](https://github.com/coderacheal/vortex/assets/33268927/7c54fb7d-d80d-40b0-9413-9b7a6712afde)